shared/cloud9.tf

```diff
@@ -0,0 +1,8 @@
+resource "aws_cloud9_environment_ec2" "shared" {
+  instance_type               = "t2.micro"
+  name                        = "team-cloud9-env"
+  automatic_stop_time_minutes = 20
+  description                 = "Shared Cloud9 instance to be used by all devs"
+  subnet_id                   = aws_subnet.public[0].id
```

Comment:
```suggestion
  subnet_id                   = aws_subnet.private[0].id
```
It looks like our old Cloud9 environments were set up on the private subnet/VPC.

Reply:
I originally had this as private VPC and private subnet but this meant the environment wasn't able to successfully be created. When I looked into it further, the github instructions all point to private VPC and public subnet
<img width="629" alt="Screenshot 2020-02-11 at 18 00 26" src="https://user-images.githubusercontent.com/47983793/74264501-a14d0080-4cf8-11ea-8fa6-9d6b3975fe78.png">

<img width="248" alt="Screenshot 2020-02-11 at 18 02 47" src="https://user-images.githubusercontent.com/47983793/74264563-c04b9280-4cf8-11ea-9f9c-c700ad00158c.png">

Although, I'd missed the differentiation between development and production values. I'll update the terraform to use the correct values for production but from what I understand, development is OK as it is.

Reply:
Actually scratch that, it should be OK for production as well:

Production VPCs/Subnets
<img width="649" alt="Screenshot 2020-02-11 at 18 05 50" src="https://user-images.githubusercontent.com/47983793/74264793-305a1880-4cf9-11ea-99af-a25ab07d3d12.png">

Reply:
Yes this looks correct then if we want it to emulate the old cloud9 setups which we do. Nice.

Reply:
I got confused because the private/public subnets are both attached to the "private" VPC. I'd thought that made them private in Terraform terms. Looking in a bit more depth though, we only use the one VPC in Terraform. The one without a name looks like a default AWS one or something 👍

Reply:
Great stuff, cheers guys 👍